cmd/liqoctl/cmd/root.go

```diff
@@ -110,16 +107,6 @@ func NewRootCommand(ctx context.Context) *cobra.Command {
 		},
 	}
 
-	// Since we cannot access internal klog configuration, we create a new flagset, let klog to install
-	// its flags, and we only set the ones we are intrested in.
-	klogFlagset := flag.NewFlagSet("klog", flag.PanicOnError)
-	klog.InitFlags(klogFlagset)
-	// These settings is required to silence the Helm library messages.
-	utilruntime.Must(klogFlagset.Set("stderrthreshold", "FATAL"))
-	klog.LogToStderr(false)
-	buffer := &bytes.Buffer{}
-	klog.SetOutput(buffer)
```

Comment:
Did you try that with the local install? There are no strange logs that were silenced by the log-disabled flags?